merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/MerlinBindings.java

```diff
@@ -175,6 +178,26 @@ private void validateActivityArguments(final Context ctx) {
     }
   }
 
+  private void validateModelArguments(final Context ctx) {
+    try {
+      final var input = parseJson(ctx.body(), hasuraMissionModelArgumentsActionP).input();
+
+      final var missionModelId = input.missionModelId();
```

Comment:
:large_orange_diamond: It seems inconsistent for one of `input`'s members to be pulled out for assignment and not the other. Maybe pull the both out first before use as done in `getActivityEffectiveArguments` etc.

Reply:
For sure, just pushed this change!